Makefile

```diff
@@ -721,8 +721,14 @@ ifneq ($(ABCREV),default)
 	$(Q) if ! (cd abc && rev="`git rev-parse $(ABCREV)`" && test "`git rev-parse HEAD`" == "$$rev"); then \
 		test $(ABCPULL) -ne 0 || { echo 'REEBE: NOP abg hc gb qngr naq NOPCHYY frg gb 0 va Znxrsvyr!' | tr 'A-Za-z' 'N-ZA-Mn-za-m'; exit 1; }; \
 		echo "Pulling ABC from $(ABCURL):"; set -x; \
-		test -d abc || git clone $(ABCURL) abc; \
-		cd abc && $(MAKE) DEP= clean && git fetch origin master && git checkout $(ABCREV); \
+		if test -d abc; then \
+			cd abc; \
+			git remote set-url origin $(ABCURL); \
+		else \
+			git clone $(ABCURL) abc; \
+			cd abc; \
+		fi; \
+		$(MAKE) DEP= clean && git fetch origin master && git checkout $(ABCREV); \
```

Comment:
This might call `make clean` on yosys if one the `cd` above fails for whatever reason.

Reply:
Good point.  I guess just adding `cd ..; \` after the `git remote` command, removing `cd abc; \` from the `else`, and reverting the `$(MAKE)` line should do the trick.

Reply:
Alternatively, `git -C abc remote ...`.

Reply:
Oooh I never knew about the `-C` option! Thanks for the suggestion; adopted in 3c0d2af.

Reply:
Me either :).  TIL.